app/models/user.rb

```diff
@@ -86,7 +86,7 @@ def moderatable
 
   #TODO: this needs tests
```

Comment:
Looks like this might be a good opportunity to resolve this TODO :grimacing: 

Also I would move the two `validates :password` lines closer together, so it's more visible how the validation rules are different for two different conditions.

Reply:
Okay let me check this as well